File: DSA/pranavdixit_590014418/Day6Question2.py
def is_strictly_increasing(nums):
    for i in range(len(nums) - 1):
        if nums[i] >= nums[i + 1]:
            return False
    return True

print("Is Increasing?", is_strictly_increasing([1, 2, 3, 4]))
print("Is Increasing?", is_strictly_increasing([1, 2, 2, 3]))
print("Is Increasing?", is_strictly_increasing([5, 4, 3]))

File: DSA/khushichaturvedi_590012789/Day_6_question_2.c
#include <stdio.h>
#include <stdbool.h>
bool isStrictlyIncreasing(int* nums, int size) {
    for (int i = 1; i < size; i++) {
        if (nums[i] <= nums[i - 1]) {
            return false; 
        }
    }
    return true; 
}
int main() {
    int nums[] = {1, 3, 5, 7, 9}; 
    int size = sizeof(nums) / sizeof(nums[0]);
    if (isStrictlyIncreasing(nums, size)) {
        printf("The array is strictly increasing.\n");
    } else {
        printf("The array is NOT strictly increasing.\n");
    }
    return 0;
}
